akka-actor/src/main/scala/akka/serialization/AsyncSerializer.scala

```diff
@@ -39,3 +46,27 @@ abstract class AsyncSerializerWithStringManifest(system: ExtendedActorSystem) ex
   }
 }
 
+/**
+ * Java API: Async serializer with string manifest that delegates synchronous calls to the asynchronous calls
+ * and blocks.
+ */
+abstract class AsyncSerializerWithStringManifestCS(system: ExtendedActorSystem) extends AsyncSerializerWithStringManifest(system) {
```

Comment:
Not that I like the `CS` naming, but we already use that in some other places. Doesn't feel worth introducing javadsl/scaladsl packages for this.